features/performance/README.md

```diff
@@ -4,7 +4,7 @@ All manifests should be applied automatically on a new cluster via `make deploy`
 
 Before you apply any feature, you should:
 
-- add `worker-rt` role to relevant nodes `oc label node <node_name> machineconfiguration.openshift.io/role=worker-rt`
+- add `worker-rt` role to relevant nodes `oc label node <node_name> node-role.kubernetes.io/worker-rt=`
```

Comment:
Just to clarify: should we add roles manually? I don't see it in deploying scripts.

Reply:
Yes, we had some discussion regarding it, and the conclusion was that possible situation when you want to apply `worker-rt` label only on a subset of nodes, so we decide to leave as a manual step.